### DISCUSSION TITLE
How can I remove the clear button of text-box on windows?

### DISCUSSION THREAD

**Discussion by taduyde** [microsoft/react-native-windows#10377] [Q&A] [answered]
On react native windows: the text-input on windows, it's show the clear button, I want to remove it. What's the way to remove the clear button?
![image](https://user-images.githubusercontent.com/8236997/183293568-4e5b996d-478d-4fce-a096-e904a59193f5.png)

**Comment by chrisglein** [comment, marked as answer]
The clear button is part of the native control and by default the TextInput control brings with it those native control assumptions. Can you clarify more why you're interested in removing the clear button and what your scenario is?

If you're interested in learning when and why it appears, there's some info in the [TextBox docs](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.TextBox?view=winrt-22621#use-textbox-for-data-input-in-a-form)

**Reply by taduyde** [reply]
Hey @chrisglein ,
in my app, I customize the password input as the style below, in the case if user click on the button the input type change to text field, so the clear button appear, it override to the eye button 
![image](https://user-images.githubusercontent.com/8236997/183807553-e2654a75-f6db-461d-b86d-3c9f3c4e07c6.png)

**Comment by alallk** [comment]
Hello, any news on how to customize/override this icon? Like this https://learn.microsoft.com/en-us/microsoft-edge/web-platform/password-reveal

**Reply by chrisglein** [reply]
Short answer is that you shouldn't have to, if the control being used maps to a UWP `PasswordBox`. See some detail [here](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.passwordbox?view=winrt-22621#password-reveal-mode), or play around with in the Win UI Gallery app.

My understanding is that in RNW when you set the [secureTextEntry](https://reactnative.dev/docs/textinput#securetextentry) on your `TextInput` that instead of using a UWP `TextBox` it uses a UWP `PasswordBox`. Which would provide this behavior by default. But that isn't the case (not sure if it's being disabled in order to better match the cross-platform expectations of `TextInput` to not include this behavior). Can you open an issue on this repo to capture this?
